runner/runner.go

```diff
@@ -32,7 +34,7 @@ const (
 )
 
 func (p ProcessState) IsDone() bool {
-	return p == UNKNOWN || p == COMPLETE || p == FAILED || p == ABORTED || p == TIMEDOUT
+	return p == COMPLETE || p == FAILED || p == ABORTED || p == TIMEDOUT
```

Comment:
Best to add a comment to that effect because I view UNKNOWN as 'I don't care what subsequent status is'.